src/Drivers/reko.config

```diff
@@ -102,6 +102,10 @@
       <Loader Extension="xml" Type="Reko.Core.TypeLibraryLoader,Reko.Core"
               Label="RekoTypeLibrary" />
       <Loader Label="RekoAmigaHeader"        Type="Reko.Environments.AmigaOS.AmigaHeaderLoader,Reko.Environments.AmigaOS" />
+
+      <!-- Python script which can be used to customize Reko decompilation process -->
+      <Loader Extension="rps" Type="Reko.Scripts.Python.PythonModule,Reko.Scripts.Python"
+              Label="RekoPythonScript" />
```

Comment:
I'm not sure if I'm comfortable with adding a file extension; is this commonplace in other contexts? Can we not keep the `.py` extension here?

Reply:
`Reko` scripts are just usual `Python` modules (with some predefined `Reko` objects). So I agree that `.py` extension looks preferable here.

Reply:
Done in 5d272c4